docs/img-src/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
I may be too picky, but the img output is not pretty enough. I imagine something slicker :)